src/components/custom/_dnumbers.tsx

```diff
@@ -1,10 +1,17 @@
-import React from 'react'
+import React, { ReactElement } from 'react'
 import styled from 'styled-components'
-import PropTypes from 'prop-types'
 import device from 'themes/device'
 import { Container, SectionContainer } from 'components/containers'
 import { Header, Text } from 'components/elements'
 
+type DNumbersProps = {
+    items?: {
+        title?: string | ReactElement
```

Comment:
Since `ReactNode` supports more possible return values of a component, it's better if we use it instead of `ReactElement`.